guide/13-managing-arcgis-applications/hub-submodule-intro.ipynb

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Please add a markdown cell at the top of the notebook that:
* is its own cell
* begins with a single harshmark to display on the website as the title of the notebook.

Current rendering has no title:
<img width="1247" alt="image" src="https://github.com/Esri/arcgis-python-api/assets/1399179/51ab4e67-53f6-4b3d-b3e6-9d0d2dbf9124">